weaviate/__init__.py

```diff
@@ -76,6 +76,15 @@
     "use_async_with_weaviate_cloud",
 ]
 
+try:
+    import weaviate_agents as agents
+
+    sys.modules["weaviate.agents"] = agents
+    __all__.append("agents")
+except ImportError:
+    pass
```

Comment:
Is this still required for the runtime while the `__init__.pyi` stubs are for the types?

Reply:
Yeah I tried without this and it didn't quite work.